### DISCUSSION TITLE
The documentation doesn't clarify if the promise api throws if status code is not 4xx or 5xx

### DISCUSSION THREAD

**Discussion by quesurifn** [sindresorhus/got#1618] [Q&A] [answered]
#### What would you like to discuss?

The documentation doesn't clarify if the promise api throws if status code is not 4xx or 5xx.

What is the behavior here? 

#### Checklist

- [X] I have read the documentation.

**Comment by szmarczak** [comment, marked as answer]
> Determines if a got.HTTPError is thrown for unsuccessful responses.

https://httpstatuses.com/

> 4×× Client Error
> 5×× Server Error
